docs/contributing/stacks.md

```diff
@@ -72,47 +72,68 @@ git remote add origin <url from github>
 git push -u origin master
 ```
 
-## Configuring Travis
-
-> NOTE: This section and the cookiecutter template should be updated to describe using GitHub
-> Actions now that jupyter/docker-stacks uses that service.
-
-Next, link your GitHub project to Travis CI to build your Docker image whenever you or someone else
-submits a pull request.
-
-1. Visit
-   [https://docs.travis-ci.com/user/getting-started/#To-get-started-with-Travis-CI](https://docs.travis-ci.com/user/getting-started/#To-get-started-with-Travis-CI)
-   and follow the instructions to add the Travis CI application to your GitHub account.
-2. Visit [https://travis-ci.org](https://travis-ci.org).
-3. Click the + symbol at the top of the left sidebar.
-   ![Travis sidebar with plus button screenshot](../_static/travis-plus-repo.png)
-4. Locate your project repository either in your primary user account or in one of the organizations
-   to which you belong.
-5. Click the toggle to enable builds for the project repository.
-6. Click the **Settings** button for that repository.
-   ![Travis enable build toggle screenshot](../_static/travis-enable-build.png)
-7. Enable **Build only if .travis.yml is present** and **Build pushed pull requests**.
-   ![Travis build settings screenshot](../_static/travis-build-settings.png)
-8. Disable **Build pushed branches**.
+## Configuring GitHub actions
+
+The cookiecutter template comes with a `.github/workflows/docker.yml` file, which allows you to use GitHub actions to build your Docker image whenever you or someone else submits a pull request.
+
+1. By default the `.github/workflows/docker.yaml` file has the following triggers configuration:
+
+   ```yaml
+   on:
+   pull_request:
+      paths-ignore:
+         - "*.md"
+         - "binder/**"
+         - "docs/**"
+         - "examples/**"
+   push:
+      branches:
+         - master
+      paths-ignore:
+         - "*.md"
+         - "binder/**"
+         - "docs/**"
+         - "examples/**"
+   ```
+
+   This will trigger the CI pipeline whenever you push to your `master` branch and when any Pull Request are made to your repository. For more details on this configurations visit the [GitHub actions documentation on triggers](https://docs.github.com/en/actions/reference/events-that-trigger-workflows).
```

Comment:
```suggestion
   This will trigger the CI pipeline whenever you push to your `main` or `master` branch and when any Pull Request are made to your repository. For more details on this configurations visit the [GitHub actions documentation on triggers](https://docs.github.com/en/actions/reference/events-that-trigger-workflows).
```

Reply:
Also `are made` -> `is made`